### PR TITLE
"validateAuthority" and "enableBroker" settings in test app profiles

### PR DESCRIPTION
**Pull request by RPangrle**

### CODE REVIEW

File: Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppSettings.m [hunk 1]
```diff
@@ -162,6 +162,10 @@ - (void)setProfileFromIndex:(NSInteger)idx
     self.redirectUri = [NSURL URLWithString:[settings objectForKey:@"redirectUri"]];
     self.resource = [settings objectForKey:@"resource"];
     self.defaultUser = [settings objectForKey:@"defaultUser"];
+    NSNumber* validate = [settings objectForKey:@"validateAuthority"];
+    self.validateAuthority = validate ? [validate boolValue] : YES;
+    NSNumber* enableBroker = [settings objectForKey:@"enableBroker"];
+    self.enableBroker = [enableBroker boolValue];
```
Comment (jasoncoolmax) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppSettings.m]: no need to be like line 166? :)
enableBroker? [enableBroker boolValue] : YES

Comment (RPangrle) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppSettings.m]: default value of enableBroker is NO, default value of validateAuthority is YES.

**Comment by RPangrle** [comment]
@jasoncoolmax Can you take another look here?

**Comment by jasoncoolmax** [comment]
:shipit:

**Pull request merged**